blog/admin.py

```diff
@@ -1,7 +1,58 @@
+from blog.models import Blog, Comment, UserVote, Profile
+
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
+
+
+class ProfileInline(admin.StackedInline):
+    model = Profile
+    can_delete = False
+    verbose_name_plural = 'Profile'
+    fk_name = 'user'
```

Comment:
nit: replace `fk` with something verbose.

Reply:
I cannot change it as it is a built-in name. I'd add an in-line comment about what it is.